src/Illuminate/Queue/Worker.php

```diff
@@ -497,6 +495,24 @@ protected function failJob($job, Throwable $e)
         return $job->fail($e);
     }
 
+    /**
+     * Calculate the backoff for the given job.
+     *
+     * @param  \Illuminate\Contracts\Queue\Job  $job
+     * @param  \Illuminate\Queue\WorkerOptions  $options
+     * @return int
+     */
+    protected function calculateBackoff($job, WorkerOptions $options)
+    {
+        $backoff = explode(',',
+            method_exists($job, 'backoff') && ! is_null($job->backoff())
+                        ? $job->backoff()
+                        : $options->backoff
+        );
+
+        return $backoff[$job->attempts() - 1] ?? last($backoff);
```

Comment:
isn't this returning a string?

Reply:
The int return type here would actually make this function return an integer as it should be returned, but no, you had to remove it 😛